addon-test-support/-private/ember-exam-qunit-test-loader.js

```diff
@@ -123,11 +123,9 @@ export default class EmberExamQUnitTestLoader extends TestLoader {
    * setupModuleMetadataHandler() register QUnit callback to enable generating module metadata file.
    */
   setupModuleMetadataHandler() {
-    this._qunit.moduleDone((metadata) => {
-      // testem:module-done-metadata is sent to server to keep track of test module details.
-      // 'metadata' contains module name, total number of assertion ran in the module, and module runtime.
```

Comment:
It would be nice to have a similar explanatory comment for testem:test-done-metadata, rather than just removing the comment.